test/e2e/util.go

```diff
@@ -2098,7 +2098,7 @@ func NewHostExecPodSpec(ns, name string) *api.Pod {
 	pod := &api.Pod{
 		TypeMeta: unversioned.TypeMeta{
 			Kind:       "Pod",
-			APIVersion: latest.GroupOrDie("").GroupVersion.Version,
```

Comment:
> In particular, this transformation is really problematic. The old code asked specifically about the "" group. The new code asks for some preferred group's version. The old code is much preferable because it's explicit what group the code is working with.

The old code did *exactly* what this code is doing now.  You simply couldn't tell that it was doing that.

The old code said: "I know that Pods live in the "" group and I need to have that group registered or I'll fail and then I want the most preferred version that was registered in that group".

The new code turns the runtime dependency into a compile-time dependency, but otherwise the logic is identical.